static/format-sdb.sh

```diff
@@ -70,10 +73,18 @@ msg_box "Formatting $SDB when you hit OK.
 
 *** WARNING: ALL YOUR DATA WILL BE ERASED! ***"
     check_command wipefs -a -f "$SDB"
```

Comment:
In the previous script I could run it as many times as I wanted. It then reformatted the drive. Could you add that here as well?

Reply:
Just noticed that the "isMounted" functions doesn't work so it doesn't recognize that the drive is mounted on a second run.

Reply:
Yes, in fact I had to modify that 
```
-if grep "$UUID" /etc/fstab
+if grep "$LABEL_" /etc/mtab
```

Reply:
sudo zfs list -pH -o mountpoint | grep -q '^/mnt/ncdata$' && echo yes

Reply:
Please check the isMounted functions as well. We need to be able to check if `/dev/sbx*` is mounted or `/mnt/ncdata` before we proceed with anything. Right now those functions are broken due to a different way of mounting the volume. 

That's why I posted the command.

Reply:
- `/dev/sda` ofc

Reply:
going to check those functions in a while... fixed the ability to correctly re-wipe everything by destroying ncdata zpool if present.